lib/features/children/add_member/repository/add_member_repository.dart

```diff
@@ -25,15 +25,19 @@ class AddMemberRepositoryImpl with AddMemberRepository {
       'allowanceType': isRGA ? 0 : 1,
     };
 
-    await apiService.addMember(body);
-    _memberAddedStreamController.add(null);
-
-    // We add this second event to the stream delayed
-    // because money-related calls require an update from Stripe for the BE
-    // which takes a bit of time
-    Future.delayed(const Duration(seconds: 2), () {
+    try {
+      await apiService.addMember(body);
+    } finally {
+      //does not get emmited if call fails
```

Comment:
What do you mean with this comment? Because the finally block will always be executed.

Reply:
oops, that was the case before, i corrected the comment should say: //always add even if there are errors